slf4j-simple/src/test/java/org/slf4j/impl/SimpleLoggerTest.java

```diff
@@ -75,4 +82,46 @@ public void loggerNameWithOneDot_NoSetLevel() {
         assertNull(simpleLogger.recursivelyComputeLevelString());
     }
 
+    @Test
+    public void givenSimpleLoggerWithListener_whenSendingParameterizedLogEvent_thenShouldNotifyListenerWithEventValues() {
+        // given
+        MockListener listener = new MockListener();
+        SimpleLogger.init(new Hashtable(), listener);
+        SimpleLogger simpleLogger = new SimpleLogger(LOGGER_NAME);
+
+        // when
+        simpleLogger.info(MESSAGE_FORMAT, PARAMETER);
+
+        // then
+        assertArrayListMatches(listener.capturedLogName, LOGGER_NAME);
+        assertArrayListMatches(listener.capturedLevel, LocationAwareLogger.INFO_INT);
+        assertArrayListMatches(listener.capturedMessageFormats, MESSAGE_FORMAT);
+        assertArrayListMatches(listener.capturedMessages, MESSAGE_FORMAT.replaceAll("\\{}", PARAMETER));
```

Comment:
I personally do not consider such value manipulation to be a good style in unit tests. Better have a constant, a'la:
```java
private static final String EXPECTED_FORMATTED_MESSAGE = "This is a format: " + PARAMETER;
```
And then:
```suggestion
        assertArrayListMatches(listener.capturedMessages, EXPECTED_FORMATTED_MESSAGE);
```